test/src/tests-persistence.ts

```diff
@@ -1507,7 +1507,7 @@ describe('persistence', () => {
             .getInstance()
             .Identity.getCurrentUser()
             .getConsentState();
-        storedConsentState.should.be.ok();
+        storedConsentState.should.be.ok;
```

Comment:
Putting the parens causes our tests to break.

Reply:
This is really weird to me still.  In `master` the test work fine.  What is the difference between the PRs?  Did some dependency change that I'm unaware of ?  https://github.com/mParticle/mparticle-web-sdk/blob/master/test/src/tests-persistence.ts#L1510

I notice that the other tests in here do use `expect`.  So if you can't figure out my question above, I suggest using `expect`, because when I put a breakpoint above this line to see what `.ok` does, it just shows a function signature, which isn't passing a test.